lib/ts/fetch.ts

```diff
@@ -421,8 +429,11 @@ export async function onUnauthorisedResponse(
                 );
 
                 logDebugMessage("onUnauthorisedResponse: Refresh call ended");
+
+                await saveTokensFromHeaders(response);
+
```

Comment:
We should think about old -> new migration path here:
- Remove old tokens like sIRTFrontend
- If method is header based, and anti-csrf token exists, we want to remove that as well.